plugins/modules/monitoring/uptimerobot.py

```diff
@@ -8,138 +8,181 @@
 
 
 DOCUMENTATION = '''
-
 module: uptimerobot
-short_description: Pause and start Uptime Robot monitoring
+short_description: Create, remove, pause and start monitors in Uptime Robot
 description:
-    - This module will let you start and pause Uptime Robot Monitoring
-author: "Nate Kingsley (@nate-kingsley)"
+    - This module lets you create, remove, pause and start monitors in Uptime Robot.
+author:
+- "Nate Kingsley (@nate-kingsley)"
+- "Ninjiner (@Ninjiner)"
 requirements:
     - Valid Uptime Robot API Key
 options:
     state:
         description:
             - Define whether or not the monitor should be running or paused.
+            - C(created) and C(absent) were added in community.general 1.3.0.
+        required: true
+        choices: [ "started", "paused", "created", "absent", "present" ]
+    url:
+        description:
+            - URL to be checked.
+        type: str
         required: true
-        choices: [ "started", "paused" ]
+        version_added: '1.3.0'
+    name:
+        description:
+            - The friendly name of the monitor.
+        type: str
+        required: true
+        version_added: '1.3.0'
+    check_type:
+        description:
+            - The kind of check, that will be performed on the url.
+        choices: [ "http", "ping" ]
+        version_added: '1.3.0'
     monitorid:
         description:
             - ID of the monitor to check.
```

Comment:
You should document when this is required.

Reply:
Changed in: https://github.com/ansible-collections/community.general/pull/961/commits/4f8e5091b2774787ce3871aac8ed1374309d0fa3